packages/okta-react/src/AuthService.js

```diff
@@ -32,12 +32,6 @@ class AuthService {
     assertClientId(authConfig.clientId);
     assertRedirectUri(authConfig.redirectUri);
 
-    // Automatically enter login flow if session has expired or was ended outside the application
-    // The default behavior can be overriden by passing your own function via config: `config.onSessionExpired`
-    if (!authConfig.onSessionExpired) {
-      authConfig.onSessionExpired = this.login.bind(this);
-    }
```

Comment:
Is there any behavior we should warn the users not to do to avoid a loop?

Reply:
Bump - I think this is good info to expose to the dev unless it is totally safe.

Reply:
Concurrency issue resolved by add pending state to login method. Tested by provide login as customized callback, works with no issue now.